internal/crypto/keystores/keystore.go

```diff
@@ -31,32 +34,59 @@ type KeyStore interface {
 	GetKey(id string) ([]byte, error)
 }
 
+// LocalKeyStore is the config value for an on-disk key store
+const LocalKeyStore = "local"
+
 // ErrUnknownKeyID is returned when the Key ID cannot be found by the keystore.
 var ErrUnknownKeyID = errors.New("unknown key id")
 
-// This structure is used by the keystore implementation to manage keys.
 type keysByID map[string][]byte
 
 // NewKeyStoreFromConfig creates an instance of a KeyStore based on the
 // AuthConfig in Minder.
 // Since our only implementation is based on reading from the local disk, do
 // all key loading during construction of the struct.
-// TODO: allow support for multiple keys/algos
-func NewKeyStoreFromConfig(config *serverconfig.AuthConfig) (KeyStore, error) {
-	key, err := config.GetTokenKey()
+func NewKeyStoreFromConfig(config serverconfig.CryptoConfig) (KeyStore, error) {
+	// You might think that a nil value cannot be deserialized to a struct,
```

Comment:
This is because the particular item is a map, and `nil` is the zero value for a map, right?  (Sorry, this comment was confusing -- maybe call out that `Config` is a map.)